apiserver/application/application_unit_test.go

```diff
@@ -252,11 +316,175 @@ func (s *ApplicationSuite) TestDestroyUnit(c *gc.C) {
 	}})
 }
 
+func (s *ApplicationSuite) setUpSeriesSubordinates() {
+	ch := mockCharm{
+		meta: &charm.Meta{
+			Series: []string{"trusty", "xenial"},
+		},
+	}
+	app := mockApplication{
+		charm:       &ch,
+		charmURL:    "cs:foo-sub",
+		subordinate: true,
+		units: []mockUnit{{
+			tag: names.NewUnitTag("foo-sub/0"),
+		}, {
+			tag: names.NewUnitTag("foo-sub/1"),
+		}},
+	}
+	s.charms["cs:foo-sub"] = &ch
+	s.applications["foo-sub"] = &app
+	s.backend.charms["cs:foo-sub"] = &ch
+	s.backend.applications["foo-sub"] = &app
+	for i, u := range s.applications["foo"].units {
+		u.subordinateNames = []string{fmt.Sprintf("foo-sub/%d", i)}
+	}
+}
+
+func (s *ApplicationSuite) TestApplicationUpdateSeries(c *gc.C) {
+	s.setUpBasicTest()
+	args := params.ApplicationUpdateSeriesArgs{
+		Args: []params.ApplicationUpdateSeriesArg{{
+			ApplicationName: "foo",
+			Series:          "trusty",
+		}, {
+			ApplicationName: "foo",
+			Series:          "zesty",
+		}},
+	}
+	results, err := s.api.UpdateApplicationSeries(args)
```

Comment:
I see that below we verify that UpdateApplicationSeries with the passed in series is called so that's ok. Ideally the test would use different application names rather than "foo" for both args though.